cwl/tools/merge_seg.cwl

```diff
@@ -15,9 +15,10 @@ arguments:
 
       egrep -v "loc.start" $(inputs.input_seg.path) |gzip >temp.gz && cat $(inputs.input_previous_merged_seg.path) temp.gz > pbta-cnv-consensus.$(inputs.biospecimen_id).seg.gz && rm temp.gz
 inputs:
-  input_seg: File
-  input_previous_merged_seg: File
+  input_seg: File?
+  input_previous_merged_seg: File?
   biospecimen_id: string
+  run_WGS_or_WXS: { type: { type: 'enum', name: run_WGS_or_WXS, symbols: ["WGS", "WXS"] } }
```

Comment:
Merged seg seems to have been set in the wf to only run when input is WGS. Shouldn't the output also be WXS?

Reply:
sorry I totally missed the comments. https://cavatica.sbgenomics.com/u/zhangb1/data-assembly-dev/tasks/f89058d6-08ad-4bc4-9263-ee8e181737d2/ here is the test for the WXS ...

Reply:
Is that intentional though? The output is:
```
 new_merged_consensus_seg No value
```
in that task output when WXS is given

Reply:
Yes. the WXS mode, won't have the consensus_seg file output, it will only use the CNVkit result